src/components/portal/portal.tsx

```diff
@@ -111,6 +112,8 @@ export class Portal {
 
     private container: HTMLElement;
 
+    private arrow: HTMLElement;
```

Comment:
I wonder if we should make this optional. So if the consumer wants to show an arrow for the popover, they can choose that. In that case, we render the arrow (and add some proper margins around the surface so that the arrow doesn't cover the trigger).

However, we would probably don't want to have this as an optional thing for the tooltip component 🤔 I guess tooltips should be always with arrows. What do you think?

Reply:
I agree 👍

Reply:
done :)